lib/matplotlib/image.py

```diff
@@ -823,44 +824,42 @@ class AxesImage(_ImageBase):
     """
     Parameters
     ----------
-    cmap : colors.Colormap
+    ax : `~.axes.Axes`
+        The axes the image will belong to.
+    cmap : str or `~matplotlib.colors.Colormap`, default: :rc:`image.cmap`
         The Colormap instance or registered colormap name used to map scalar
         data to colors.
-    norm : colors.Normalize
+    norm : `~matplotlib.colors.Normalize`
         Maps luminance to 0-1.
-    interpolation : str
+    interpolation : str, default: :rc:`image.interpolation`
         Supported values are 'none', 'antialiased', 'nearest', 'bilinear',
         'bicubic', 'spline16', 'spline36', 'hanning', 'hamming', 'hermite',
         'kaiser', 'quadric', 'catrom', 'gaussian', 'bessel', 'mitchell',
         'sinc', 'lanczos'.
-    origin : {'upper', 'lower'}
+    origin : {'upper', 'lower'}, default: :rc:`image.origin`
         Place the [0, 0] index of the array in the upper left or lower left
         corner of the axes. The convention 'upper' is typically used for
         matrices and images.
-    extent : tuple
+    extent : tuple, optional
```

Comment:
I thought the policy was to omit 'optional' now since one can check the function signature to see if it's optional?